annotation/tests/test_distribution.py

```diff
@@ -1905,3 +1924,25 @@ def test_redistribute_error(
     )
     assert exc_info.value.status_code == 500
     assert "Connection timeout" in str(exc_info.value.detail)
+
+
+def test_create_partial_validation_tasks():
+    validation_files_pages = {1: list(range(1, 81))}
+    validation_tasks = []
+    with patch(
+        "annotation.distribution.main.filter_validation_files_pages",
+        return_value=validation_files_pages,
+    ), patch("annotation.distribution.main.SPLIT_MULTIPAGE_DOC", True):
+        create_partial_validation_tasks(
+            {},
+            [{"user_id": "user1"}, {"user_id": "user2"}],
+            {},
+            1,
+            validation_tasks,
+            TaskStatusEnumSchema.pending,
+            None,
+        )
+
+        assert len(validation_tasks) == 2
+        assert validation_tasks[0]["file_id"] == 1
+        assert len(validation_tasks[0]["pages"]) == 50
```

Comment:
Since we are using `SPLIT_MULTIPAGE_DOC` flag, we should assert both of the validation tasks and convey the splitting.
`assert len(validation_tasks[1]["pages"]) == 30` or even `assert validation_tasks[1]["pages"] == list(range(51,81))`